docs/resources/aws_config_delivery_channel.md

```diff
@@ -0,0 +1,77 @@
+---
+title: About the aws_config_delivery_channel Resource
+---
+
+# aws_config_delivery_channel
+
+Use the `aws_config_delivery_channel` InSpec audit resource to test properties of your aws config service.
+
+<br>
+
+## Syntax
+
+An `aws_config_delivery_channel` resource block declares the tests for a single AWS configuration delivery channel.
+
+    describe aws_config_delivery_channel('my_channel') do
+      it { should exist }
+    end
+
+    describe aws_config_delivery_channel(recorder_name: 'my-channel') do
+      it { should exist }
+    end
+
+<br>
+
+## Examples
+
+The following examples show how to use this InSpec audit resource.
+
+### Test if the recorder is active and recording.
+
+    describe aws_config_delivery_channel(recorder_name: 'my-recorder') do
+      it { should be_recording }
+    end
+    
+## Properties
+    
+### s3_bucket_name
```

Comment:
Might clarify that s3_bucket_name is an optional value - a delivery channel can talk to SNS, S3, or both.  That info could also go at the top of the doc, where we describe the resource.